repo/packages/N/neo4j-proxy/0/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "packagingVersion": "3.0",
+  "name": "neo4j-proxy",
+  "version": "1.0.0",
```

Comment:
`version` should be `neo4jProxyVersion-packagingVersion`. 
Ex: if neo4jProxyVersion is 1.0.0 then `version` would be `1.0.0-0.0.1`